python/cuda_parallel/pyproject.toml

```diff
@@ -21,6 +21,9 @@ dependencies = [
   "numba>=0.60.0",
   "cuda-python==12.*",
   "cupy-cuda12x",
+  "nvidia-cuda-runtime-cu12",
```

Comment:
Q: do we need cudart headers? If not, I suppose this can be removed now that we switch to `CUDA::cudart_static`

Reply:
Good question, I'll try it out: commit ce889d5c419bd8330b2c58e5363f92c118b7df03